prow/pod-utils/downwardapi/jobspec.go

```diff
@@ -134,13 +136,17 @@ func EnvForSpec(spec JobSpec) (map[string]string, error) {
 
 	env[pullNumberEnv] = strconv.Itoa(spec.Refs.Pulls[0].Number)
 	env[pullPullShaEnv] = spec.Refs.Pulls[0].SHA
+
+	if changes := envFromPullChangedFiles(spec.Refs.Pulls); changes != "" {
```

Comment:
This function is only called for `presubmits`, we need to also provide a value for batch jobs (currently Tide sets the field, but this code will ignore it). I would expect the value for batches to be the set union of the files changed by each PR.

Reply:
Thank you for the review. This is pending #13731 (to use git client in Tide). But I have updated the PR to address this, i.e. add reference to pull requests' changed files in batch jobs.